web-app/src/components/FlashCard.spec.js

```diff
@@ -42,11 +48,11 @@ describe("the FlashCard component", () => {
   });
 });
 
-const renderFlashCard = () => {
+const renderFlashCard = (clickable = true) => {
   const flashCard1 = {
     title: "title 1",
     body: "text 1",
     image: "image url",
   };
-  return render(<FlashCard card={flashCard1} />);
+  return render(<FlashCard clickable={clickable} card={flashCard1} />);
```

Comment:
does this need to be set for the value cards or part of a different ticket?

Reply:
yeah that needs to be part of value cards to disable the + button